content/realtime/messages.textile

```diff
@@ -481,7 +481,7 @@ blang[jsall].
   For example, if you wanted to send an emoji reaction to the previous message above you could send something such as the following:
 
   bc[jsall]. function sendReaction(emoji) {
-    channel.publish('event_name', { body: emoji, extras: { ref: { type: "com.ably.reaction", timeserial: "1656424960320-1" } } })
+    channel.publish({ data: emoji, extras: { ref: { type: "com.ably.reaction", timeserial: "1656424960320-1" } } })
```

Comment:
(might be worth adding a `name: 'event_name'` to the object, to make that clearer? shrug)